language/src/modules/Language.rb

```diff
@@ -1402,6 +1402,25 @@ def fbiterm?
       Builtins.getenv("TERM") == "iterm"
     end
 
+    # Returns the command to make language settings persistent
+    #
+    # It is different depending on the stage, since for a not installed system it is needed to use
+    # the `systemd-firsrboot` tool, which does not work in a chroot
+    #
+    # @param locale [String]
+    #
+    # @return [Array] an array containing the command to be executed
+    def save_command_for(locale)
```

Comment:
NP: I am not sure if method name is good. IT is a bit misleading for me. Maybe something like `store_locale_command` sounds better for me. But just np.

Reply:
Changed ;)